### DISCUSSION TITLE
MongoDB?

### DISCUSSION THREAD

**Discussion by ghost** [webdev-20/shorti-url-shortener#101] [Q&A]
How are people using MongoDB for this project, in development?
I assume mongodb atlas or a local mongodb instance?

If so, would it be a good addition to add a ticket to include docker-compose.yml file for running mongodb instance in docker?

**Comment by cherylli** [comment]
we are using mongo atlas. not sure about running the instance locally, unless it's only for development

**Comment by ghost** [comment]
Gotcha. When I went to run the project, I saw it needed MongoDB. Usually, when I need to use MongoDB, I set it up through docker, only for local development. This only requires one docker-compose.yml file I believe.

I think that if others work on the project, they might find this dockerized mongodb setup useful if they wish to not use mongodb atlas for local development. 

I can make a github issue detailing this further, and work on it. If this is something that is not wanted in the project, then feel free to let me know or close the issue
